apple/internal/run_support.bzl

```diff
@@ -37,7 +37,25 @@ def _register_simulator_executable(ctx, output):
         },
     )
 
+def _register_macos_executable(ctx, output):
+    """Registers an action that runs the bundled macOS app.
+
+    Args:
+      ctx: The Skylark context.
+      output: The `File` representing where the executable should be generated.
+    """
+    ctx.actions.expand_template(
+        output = output,
+        is_executable = True,
+        template = ctx.file._macos_runner_template,
+        substitutions = {
+            "%app_name%": ctx.label.name,
```

Comment:
sorry, it must have not saved... you'll need to use `bundling.bundle_name(ctx)`  here in order to get the bundle name as as configured in the `bundle_name` attribute.

Reply:
@sergiocampama - No worries! Just updated, PTALe